README.md

```diff
@@ -21,15 +21,15 @@ The Forms Community Toolkit is a collection of helper functions, custom controls
 | Gestures | dev | [![Build status](https://ci.appveyor.com/api/projects/status/yjymobl63rs8qfau/branch/dev?svg=true)](https://ci.appveyor.com/project/FormsCommunityToolkit/gestures/branch/dev) | |
 
 ## Contributing
-Do you want to contribute? Here are our [contribution guidelines](https://github.com/FormsCommunityToolkit/FormsCommunityToolkit/blob/master/contributing.md).
+Do you want to contribute? Here are our [contribution guidelines](https://github.com/xamarin/XamarinCommunityToolkit/blob/master/contributing.md).
 
 ## Getting started
 
 Coming Soon.
 
-## Forms Community Toolkit Sample App
+## Xamarin Community Toolkit Sample App
 
-Want to see the toolkit in action before jumping into the code?  Download and play with the [Forms Toolkit Sample App]().
+Want to see the toolkit in action before jumping into the code?  Download and play with the [Sample App]().
```

Comment:
Is there an app we can link to?

Reply:
There is an inline sample app ( as in the repo ) but not one in the store yet.

Reply:
yeh, that's what the other readme links to, I'll update this to /Samples